src/identity/components/GlobalHeader/GlobalHeaderStyle.ts

```diff
@@ -0,0 +1,12 @@
+export const globalHeaderStyle = {
```

Comment:
Tried using `scss` and setting className, but it looks like the classes aren't overriding in the clockface component, so reverted to a file that holds the variables for setting `style`.